modules/outputs.tf

```diff
@@ -0,0 +1,34 @@
+output "rcb_vm_name" {
+  value       = module.rcb_vm.name
```

Comment:
Treaba asta nu merge in terraform, daca ai nevoie sa folosesti in main.tf un output o sa scrii ceva de genul
`name =  module.rcb_vm.name`
Nu e nevoie sa le definesti iar, cred ca poti sa stergi tot fisierul asta